Lib/glyphsLib/builder/features.py

```diff
@@ -138,7 +138,11 @@ def _to_ufo_features(
                     # Replace special chars backslash and doublequote for AFDKO syntax
                     name = name.replace("\\", r"\005c").replace('"', r"\0022")
                     feature_names = ["featureNames {", f'  name "{name}";', "};"]
-        elif font.format_version == 3 and feature.labels:
+        elif (
+            font.format_version == 3
```

Comment:
do we need to check the format_version? if there are any feature.labels we just use them..